src/__tests__/api/xlsx/sep-xlsx.spec.ts

```diff
@@ -7,23 +7,25 @@ import app from '../../../app';
 import { generateTmpPath } from '../../../util/fileSystem';
 import testPayloads from '../../fixtures/xlsx-payloads.json';
 
-beforeAll(done => {
-  setTimeout(done, 5000);
+beforeAll(() => {
```

Comment:
Couldn't this be just like this?
`beforeAll(() => {}, 15000);`

Reply:
Otherwise in Node 16 there is now
```
import { setTimeout } from 'timers/promises';

await setTimeout(5000)
```

Reply:
It might be possible I will try it. Thanks for pointing this out.

Reply:
@jekabs-karklins I updated this and now it is using 
`
    beforeAll(async () => {
       await setTimeout(5000);
     }, 10000);
`